packages/css/test/no-babel/__snapshots__/index.test.js.snap

```diff
@@ -231,8 +231,3 @@ exports[`css random expressions undefined return 1`] = `
   hello world
 </h1>
 `;
-
-exports[`css styled throws a nice error when using the styled shorthand without babel-plugin-emotion 1`] = `
```

Comment:
why this has been removed?

Reply:
Because this doesn't exist anymore(there used to be a proxy in dev and it caused weird problems in some cases) and IIRC the test was skipped